docker-compose-files/openshift-compose/docker-compose.yml

```diff
@@ -0,0 +1,143 @@
+version: "2"
+services:
+  hello:
+    image: surajd/hello
+    ports:
+    - "8080"
+    - "8778"
+    - "8443"
+    labels:
+     - "app=hello"
+     - "hystrix.enabled=true"
+
+  hola:
+    image: containscafeine/hola
+    ports:
+    - "8080"
+    labels:
+    - "app=hola"
+    - "hystrix.enabled=true"
+
+  aloha:
+    image: surajd/aloha
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "app=aloha"
+    - "hystrix.enabled=true"
+
+  ola:
+    image: surajd/ola
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "app=ola"
+    - "hystrix.enabled=true"
+
+  namaste:
+    image: surajd/namaste
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "app=namaste"
+    - "hystrix.enabled=true"
+
+  bonjour:
+    image: surajd/bonjour
+    ports:
+    - "8080"
+    labels:
+    - "app=bonjour"
+    - "hystrix.enabled=true"
+
+  api-gateway:
+    image: surajd/api-gateway
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "app=api-gateway"
+    - "hystrix.enabled=true"
+
+  hystrix-dashboard:
+    image: fabric8/hystrix-dashboard:1.0.15
+    ports:
+    - "8080"
+    labels:
+    - "project=hystrix-dashboard"
+    - "provider=fabric8"
+    - "version=1.0.15"
+    - "group=io.fabric8.kubeflix"
+    - "package=kubeflix"
+    environment:
+      - KUBERNETES_NAMESPACE=helloworld-msa
+
+
+  turbine-server:
+    image: fabric8/turbine-server:1.0.15
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "project=turbine-server"
+    - "provider=fabric8"
+    - "version=1.0.15"
+    - "group=io.fabric8.kubeflix"
+    - "package=kubeflix"
+    environment:
+      - KUBERNETES_NAMESPACE=helloworld-msa
+
+
+  zipkin-mysql:
+    image: fabric8/zipkin-mysql:0.0.8
+    ports:
+    - "3306"
+    labels:
+    - "project=zipkin-mysql"
+    - "provider=fabric8"
+    - "version=0.0.8"
+    - "group=io.fabric8.zipkin"
+
+  zipkin-query:
+    image: openzipkin/zipkin-query:1.39.3
+    ports:
+    - "9411"
+    labels:
+    - "project=zipkin-query"
+    - "provider=fabric8"
+    - "version=0.0.8"
+    - "group=io.fabric8.zipkin"
+    environment:
+      - STORAGE_PORT_3306_TCP_ADDR=zipkin-mysql:3306
+      - STORAGE_TYPE=mysql
+      - TRANSPORT_TYPE=http
+
+  zipkin-query-admin:
+    image: openzipkin/zipkin-query:1.39.3
+    ports:
+    - "9901"
+    labels:
+    - "project=zipkin-query"
+    - "provider=fabric8"
+    - "version=0.0.8"
+    - "group=io.fabric8.zipkin"
+    environment:
+      - STORAGE_PORT_3306_TCP_ADDR=zipkin-mysql:3306
+      - STORAGE_TYPE=mysql
+      - TRANSPORT_TYPE=http
+  
+  frontend:
+    image: surajd/frontend:hystrixport
+    links:
+    - hello
+    ports:
+    - "8080"
+    labels:
+    - "app=frontend"
+    - "hystrix.enabled:true"
```

Comment:
You made me realize that the frontend doesn't support hystrix. I just removed it in this commit: https://github.com/redhat-helloworld-msa/helloworld-msa/commit/7e21b725611cb5724f1c2a823f91e93ba0426f6d - You should also remove this label